types/json/binary_test.go

```diff
@@ -67,6 +70,10 @@ func TestBinaryJSONExtract(t *testing.T) {
 		// test extract with multi path expressions.
 		{bj1, []string{"$.a", "$[5]"}, mustParseBinaryFromString(t, `[[1, "2", {"aa": "bb"}, 4.0, {"aa": "cc"}]]`), true, nil},
 		{bj2, []string{"$.a", "$[0]"}, mustParseBinaryFromString(t, `[{"a": 1, "b": true}]`), true, nil},
+		{bj3, []string{`$.properties.$type`}, mustParseBinaryFromString(t, `"TiDB"`), true, nil},
```

Comment:
L73~L76 is not the `multi path expressions.` please move this test to L69 and maybe add some `multi path` test case for `$`

Reply:
I've moved these tests and add multi path test case for $